packages/number-field/src/number-field.css

```diff
@@ -53,3 +52,52 @@ governing permissions and limitations under the License.
         0
     );
 }
+
+:host([invalid]:not([hide-stepper])) #textfield .icon {
+    --mod-textfield-icon-spacing-inline-end-invalid: calc(
+        var(--system-infield-button-top-width) +
```

Comment:
wasn't sure if there was a better token to use for the width of the stepper buttons to use in the calc